Assertion `ret_trits->data[i] == 0' failed on ARM64

I managed to build dcurl on Cavium ThunderX (aarch64) machine. However, it failed to pass the test suite:
```
$ make BUILD_STAT=1 check
...
*** Validating build/test-dcurl ***
test-dcurl: tests/test-dcurl.c:67: main: Assertion `ret_trits->data[i] == 0' failed.
Aborted (core dumped)
mk/common.mk:27: recipe for target 'build/test-dcurl.done' failed
```

--- a/src/pow_c.c
+++ b/src/pow_c.c
@@ -121,7 +121,7 @@ static long long int loop_cpu(uint64_t *lmid,
             return i * 64;
         }
     }
-    return -i * 64 + 1;
+    return -i * 64 - 1;
 }
 
 static void para(int8_t in[], uint64_t l[], uint64_t h[])

